UnityPy/files/BundleFile.py

```diff
@@ -523,6 +523,14 @@ def decompress_data(
     def get_version_tuple(self) -> Tuple[int, int, int]:
         """Returns the version as a tuple."""
         version = self.version_engine
-        if not version or version == "0.0.0":
-            version = config.get_fallback_version()
-        return tuple(map(int, reVersion.match(version).groups()))
+        match = None
+        if version and version != "0.0.0":
+            match = reVersion.match(version)
+            if not match or len(match.groups()) < 3:
+                match = None
```

Comment:
In this case it might actually be better to throw an error if the fallback version isn't set.

Reply:
....which requires the fallback version to not be set by default...which would be better than using an old version as of now anyway.

Reply:
I can also just change that with a follow up commit, so it can stay this way.